lib/ecto/query.ex

```diff
@@ -1752,12 +1752,20 @@ defmodule Ecto.Query do
   "nulls first" or "nulls last" is specific to each database implementation.
 
   `order_by` may be invoked or listed in a query many times. New expressions
-  are always appended to the previous ones.
+  can be appended or preprended to the existing ones. The behaviour is controlled
+  through the `:position` option. By default, new expressions are appended.
 
   `order_by` also accepts a list of atoms where each atom refers to a field in
   source or a keyword list where the direction is given as key and the field
   to order as value.
 
+  ## Options
+
+    * `:position` - where to place the order expression relative to the
```

Comment:
I think `:position` is a bit too ambiguous, given we already have positional bindings and so on. Maybe "placement"? "mode"?

Reply:
`mode` sounds good to me. done!